### DISCUSSION TITLE
New Themes Feedback

### DISCUSSION THREAD

**Discussion by mnmalist** [zed-industries/zed#18650] [Show and tell]
I started creating two themes for Zed:

1. A macOS inspired window theme, with syntax theme inspired by Xcode and Nova

![mTheme](https://github.com/user-attachments/assets/5a319fad-8a87-40a8-bad7-d0c516f35f4a)

2. A Vercel inspired window/syntax theme

![Triangle](https://github.com/user-attachments/assets/a8535030-016b-463e-a266-41dc12c4bee0)

What are your thoughts on these? How can I improve them?

Would you like to see them make it to the Zed Extensions?

**Comment by mnmalist** [comment]
Also, would you like them to have light modes, as well?

**Reply by GrigoriiPrudnikov** [reply]
bro, send your extension to me, when u finish, please

**Reply by mnmalist** [reply]
Deal!

**Comment by GrigoriiPrudnikov** [comment]
I think vercel theme is very good!

**Comment by widersky** [comment]
Both looking great!
